plugins/inputs/win_perf_counters/win_perf_counters.go

```diff
@@ -135,20 +143,25 @@ func (m *Win_PerfCounters) SampleConfig() string {
 func (m *Win_PerfCounters) ParseConfig() error {
 	var query string
 
+	if m.UseWinTimestamp {
+		PdhUseWinTimestamps()
+	}
+
 	if len(m.Object) > 0 {
 		for _, PerfObject := range m.Object {
 			for _, counter := range PerfObject.Counters {
 				for _, instance := range PerfObject.Instances {
 					objectname := PerfObject.ObjectName
+					nodename := PerfObject.NodeName
 
 					if instance == "------" {
-						query = "\\" + objectname + "\\" + counter
+						query = "\\\\" + nodename + "\\" + objectname + "\\" + counter
```

Comment:
I can tell you now. The node name is required to be specified. I personally like this, because the measurement gets tagged with the node name, which is now different from the collecting host, but it's not backwards compatible.

Let me check a different idea in which would be backwards compatible.